app/views/users/_no_targets_created.html.haml

```diff
@@ -0,0 +1,19 @@
+.middle-section
+  .no-targets-section
+    %h4
+      = I18n.t(:create_first_target)
+    %p
+      = I18n.t(:most_popular_targets)
+    .popular-topics
+      .topic-div
```

Comment:
In order to avoid code duplication, you could do something like this:
```
- %w(Football Travel Music).each do |topic|
  .topic-div
    .topic-img
      = image_tag "#{topic}.png"
    = topic
```